source/connect.txt

```diff
@@ -204,6 +204,21 @@ For ``tls`` connections:
 
         mongosh "mongodb://mongodb0.example.com:28015" --tls
 
+Connect to a Specific Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a specific database:
+
+- You can specify a database in the URL path when passing a connection
+  string. The database in the URL path is automatically selected unless
+  you specify a different database.
+
+   .. example::
+
+     .. code-block:: sh
+
+         mongosh 'mongodb://localhost:27017/db1' 
```

Comment:
Can we add a sentence below this code block (but still in the example block) what this does? When we include code, we should try to provide some sort of context.

Something like: "The following connection string URI connects to the `db1` database".